tests/externs.el

```diff
@@ -59,3 +59,47 @@
 ;;;TODO
 ;; ensure that any symbols bound with the import syntax are added to the extern list
 ;; ensure that any symbols bound with the export syntax exist in the file scope
+
+;; Test `/*jslint*/` declarations
+
+(ert-deftest js2-finds-jslint-declaration-externs ()
+  (with-temp-buffer
+    (insert "/*jslint browser, node:false, es6: true */")
```

Comment:
OK. But `node: true` works? And is the same as just `node`?

The documentation examples look like

```js
/*jslint
    es6, maxerr: 10, node
*/
```